README.md

```diff
@@ -34,6 +20,8 @@ See our [examples](examples) for more information on including this library in y
 - [Maven - pom.xml](./examples/maven-example/pom.xml)
 - [Gradle - build.gradle](./examples/gradle-example/build.gradle)
 
+> Are you using a different build tool other than Maven or Gradle? Or, are you interested in just browsing what is available in our repository? Or, do you just need to download `tmc-auth`'s required jar files directly? You can find everything you need in our [Cloudsmith](https://cloudsmith.io/~autonomic/repos/au-tmc-oss/groups/) repository.
```

Comment:
```suggestion
> Are you using a different build tool other than Maven or Gradle? Are you interested in just browsing what is available in our repository? Do you just need to download `tmc-auth`'s required jar files directly? You can find everything you need in our [Cloudsmith](https://cloudsmith.io/~autonomic/repos/au-tmc-oss/groups/) repository.
```

I think you can remove the `Or,` based on this article:
https://www.grammarly.com/blog/comma-before-or/